### ISSUE TITLE
Remove ul and ol from examples

### ISSUE THREAD

**Issue by mattiapontonio** [mdn/sprints#3962]
## URL(s)
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

## Request type

- [ ] New documentation
- [X] Correction or update

### Result
![Screenshot_20201212-223036.png](https://user-images.githubusercontent.com/44380480/101995435-121ec980-3cca-11eb-91ff-bf3e0fc0ce70.png)

**Comment by chrisdavidmills** [comment]
Nah. Even though it might seem like "too much markup", the examples are showing lists of links, so using a list is a semantic improvement on just multiple anchor elements.

**Comment by mattiapontonio** [comment]
@chrisdavidmills 

It could be argued that a list should be arranged vertically perhaps with dots that identify an element on the total or a number if the order is important.

![Screenshot_20201213-121739.png](https://user-images.githubusercontent.com/44380480/102010256-42ec1680-3d3d-11eb-96ab-9fa6c324317d.png)

🙂
